clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -158,13 +161,33 @@ private ThreadPoolExecutor initHelixStateTransitionThreadPool(int size, String t
     return helixStateTransitionThreadPool;
   }
 
+  public HelixManagerProperty buildHelixManagerProperty(VeniceServerConfig config) {
+    InstanceConstants.InstanceOperation instanceOperation;
+    if (config.getHelixJoinAsUnknown()) {
+      instanceOperation = InstanceConstants.InstanceOperation.UNKNOWN;
+    } else {
+      instanceOperation = InstanceConstants.InstanceOperation.ENABLE;
+    }
```

Comment:
When not specified explicitly, can we skip setting any value and keep the code same as current?